tls/s2n_handshake.h

```diff
@@ -52,6 +52,9 @@ struct s2n_handshake_parameters {
 
     /* Signature/hash algorithm pairs offered by the server in the certificate request */
     struct s2n_sig_hash_alg_pairs server_sig_hash_algs;
+
+    /* The cert chain we will send the peer. */
+    struct s2n_cert_chain_and_key *chain_and_key;
```

Comment:
What do you think about making this 'own_chain_and_key' or something else similar to signify it is the sender's and not the peer's?